src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -535,7 +535,7 @@ public List<OffenderPropertyContainer> getActivePropertyContainers() {
 
     public List<OffenderProfileDetail> getActiveProfileDetails() {
         return profileDetails.stream()
-            .filter(pd -> Objects.nonNull(pd.getCode()))
```

Comment:
If a profile value is a number then it's not a PROFILE_CODE so this is null and the value is stored in propert profileCode